From 5c007bd0d99c45a0fb6fc042f451e06350fbe328 Mon Sep 17 00:00:00 2001
From: from minimyth2 <unknown@unknown.org>
Date: Thu, 18 Jan 2024 08:24:03 +0100
Subject: [PATCH 137/184] 
 0612-gpu-drm-panfrost-panfrost-devfreq-revert-set-opp-to-the-recommended.patch

---
 drivers/gpu/drm/panfrost/panfrost_devfreq.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/drivers/gpu/drm/panfrost/panfrost_devfreq.c b/drivers/gpu/drm/panfrost/panfrost_devfreq.c
index 5a65c24b9..b01850c37 100644
--- a/drivers/gpu/drm/panfrost/panfrost_devfreq.c
+++ b/drivers/gpu/drm/panfrost/panfrost_devfreq.c
@@ -131,17 +131,6 @@ int panfrost_devfreq_init(struct panfrost_device *pfdev)
 		return PTR_ERR(opp);
 
 	panfrost_devfreq_profile.initial_freq = cur_freq;
-
-	/*
-	 * Set the recommend OPP this will enable and configure the regulator
-	 * if any and will avoid a switch off by regulator_late_cleanup()
-	 */
-	ret = dev_pm_opp_set_opp(dev, opp);
-	if (ret) {
-		DRM_DEV_ERROR(dev, "Couldn't set recommended OPP\n");
-		return ret;
-	}
-
 	dev_pm_opp_put(opp);
 
 	/*
-- 
2.39.2

